### PR TITLE
MS-724 What are your contact details? Page

### PR DESCRIPTION
**Pull request by jjrobothamho**
What are your contact details? Page + acceptance tests

### CODE REVIEW

File: acceptance-test/user-pathways/happy-path/happy-path.test.js [hunk 1]
```diff
@@ -195,9 +198,21 @@ describe.only('User path(s)', () => {
         // supporting-documents-add
         await clickSelector(page, UPLOAD_DOCUMENT_PAGE_2_NO_OPTION);
         await clickSelector(page, CONTINUE_BUTTON);
-        // Run through the skeleton until we reach the end
+        // fr-details
+        await page.waitForSelector(FR_DETAILS_NAME_INPUT);
+        await page.$eval(FR_DETAILS_NAME_INPUT, (element) => {
+            element.value = 'Test';
```
Comment (sulthan-ahmed) [inline acceptance-test/user-pathways/happy-path/happy-path.test.js]: I'd put in a real name like 'Mo' or 'Ahmed'

File: acceptance-test/user-pathways/happy-path/happy-path.test.js [hunk 2]
```diff
@@ -195,9 +198,21 @@ describe.only('User path(s)', () => {
...
+        });
+        await page.$eval(FR_DETAILS_ROLE_INPUT, (element) => {
+            element.value = 'Test Role';
```
Comment (sulthan-ahmed) [inline acceptance-test/user-pathways/happy-path/happy-path.test.js]: same as above

File: apps/nrm/translations/src/en/pages.json [hunk 3]
```diff
@@ -96,7 +96,10 @@
     "list-header": "Documents added"
   },
   "fr-details": {
-    "header": "Placeholder page"
+    "header": "What are your contact details?",
+    "read-only-field-1-label": "Email",
+    "read-only-field-1-hint": "This is the address you gave at the start of the form.",
+    "read-only-field-1-placeholder": "FR EMAIL PLACEHOLDER"
```
Comment (sulthan-ahmed) [inline apps/nrm/translations/src/en/pages.json]: are you going to do this later?

File: acceptance-test/user-pathways/upload-file/upload-file.test.js [hunk 4]
```diff
@@ -203,9 +206,21 @@ describe('Upload File(s)', () => {
...
+        // fr-details
+        await page.waitForSelector(FR_DETAILS_NAME_INPUT);
+        await page.$eval(FR_DETAILS_NAME_INPUT, (element) => {
+            element.value = 'Test';
+        });
+        await page.$eval(FR_DETAILS_ROLE_INPUT, (element) => {
+            element.value = 'Test Role';
+        });
+        await page.$eval(FR_DETAILS_PHONE_INPUT, (element) => {
+            element.value = '00000000000';
```
Comment (sulthan-ahmed) [inline acceptance-test/user-pathways/upload-file/upload-file.test.js]: same as above, why is it repeated? different tests?

File: acceptance-test/user-pathways/happy-path/happy-path.test.js (hunk 1 above)
Comment (jjrobothamho) [reply acceptance-test/user-pathways/happy-path/happy-path.test.js]: No worries I added them in another PR I'll copy them across

File: apps/nrm/translations/src/en/pages.json (hunk 3 above)
Comment (jjrobothamho) [reply apps/nrm/translations/src/en/pages.json]: Yes In MS-826 , We are we getting those values from? @sulthan-ahmed

File: acceptance-test/user-pathways/happy-path/happy-path.test.js (hunk 1 above)
Comment (sulthan-ahmed) [reply acceptance-test/user-pathways/happy-path/happy-path.test.js]: Note: it doesn't need to be Mo or Ahmed 😄

File: apps/nrm/translations/src/en/pages.json (hunk 3 above)
Comment (sulthan-ahmed) [reply apps/nrm/translations/src/en/pages.json]: it'll be in the sessionModel something like `req.sessionModel.get('email')` We do this in right-to-rent-check somewhere

**Pull request merged**